cli/cmd/root.go

```
@@ -62,7 +66,7 @@ func initConfig() {
 			os.Exit(1)
 		}
 
-		// Search config in home directory with name ".cobalt" (without extension).
+		// Search config in home directory with name ".orion" (without extension).
 		viper.AddConfigPath(home)
 		viper.SetConfigName(cfgFileName)
 	}
```

we should raise the exception if we falter on `viper.ReadInConfig()`

---

I actually removed it because we're not using it at this point... I've added a comment TODO if we start using viper